### PR TITLE
Add supported models to the capabilities response

### PR DESCRIPTION
**Pull request by ray-milkey**

### CODE REVIEW

File: pkg/northbound/gnmi/v2/service.go [hunk 1]
```diff
@@ -79,15 +79,17 @@ type Server struct {
...
-	// TODO adds capability
-	/*capabilities, err := s.modelRegistry.Capabilities()
-	if err != nil {
-		return nil, err
-	}*/
+	plugins := s.pluginRegistry.GetPlugins()
+
+	supportedModels := make([]*gnmi.ModelData, 0)
+	for _, plugin := range plugins {
+		capabilities := plugin.Capabilities(ctx, nil)
```
Comment (adibrastegarnia) [inline pkg/northbound/gnmi/v2/service.go:86]: What is the second argument here which can be nil?

Comment (ray-milkey) [reply pkg/northbound/gnmi/v2/service.go:86]: Not sure what the intention is - in the function itself the parameter is unused

Comment (adibrastegarnia) [reply pkg/northbound/gnmi/v2/service.go:86]: OK perhaps we should look into what it was before and remove or complete it if it is needed. Going to approve it that you can merge.

**Review by adibrastegarnia — APPROVED** [review]

File: pkg/northbound/gnmi/v2/service.go (hunk 1 above)
Comment (ray-milkey) [reply pkg/northbound/gnmi/v2/service.go:86]: OK. I'll find out of we can remove that parameter before I merge it.

**Review by tomikazi — APPROVED** [review]

**Pull request merged**